Integrations/Tenable_sc/Tenable_sc.py

```diff
@@ -1543,10 +1551,10 @@ def list_alerts_command():
         'Name': a['name'],
         'State': 'Triggered' if a['didTriggerLastEvaluation'] == 'true' else 'Not Triggered',
         'Actions': demisto.dt(a['action'], 'type'),
-        'LastTriggered': datetime.utcfromtimestamp(int(a['lastTriggered'])).strftime(
-            '%Y-%m-%dT%H:%M:%SZ') if a['lastTriggered'] and int(a['lastTriggered']) > 0 else 'Never',
-        'LastEvaluated': datetime.utcfromtimestamp(int(a['lastEvaluated'])).strftime(
-            '%Y-%m-%dT%H:%M:%SZ') if a['lastEvaluated'] else '',
+        'LastTriggered': timestamp_to_utc(a['lastTriggered'],
+                                          additional_cond=(int(a['lastTriggered']) > 0),
+                                          default_returned_value='Never'),
```

Comment:
Since this "default_returned_value" argument is only used within this function, I think it should be discarded as an argument and replaced with an inner check or some other more readable solution